protocols/ping/src/handler.rs

```diff
@@ -183,6 +183,7 @@ impl Handler {
         >,
     ) {
         self.outbound = None; // Request a new substream on the next `poll`.
+        self.interval.reset(Duration::new(0, 0));
```

Comment:
Might be worth to document why we are resetting it upon an error.

Reply:
Done